api/info.go

```diff
@@ -0,0 +1,31 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/puppetlabs/wash/journal"
+)
+
+// TODO: Include the entry's full metadata?
```

Comment:
🤷‍♂️ that seems a little weird when we also have `/fs/meta`. Would definitely leave it out for now.